cli/src/test/java/keywhiz/cli/commands/AddActionTest.java

```diff
@@ -52,10 +52,13 @@
   AddActionConfig addActionConfig;
   AddAction addAction;
 
-  Client client = new Client(4, "newClient", null, null, null, null, null, null, null, true, false, null);
+  Client client =
+      new Client(4, "newClient", null, null, null, null, null, null, null, true, false, null);
```

Comment:
I'm sure the code here is functional (and looked like this beforehand, so no issues were introduced in this PR), but aren't there a lot of arguments for this constructor?

Reply:
It definitely is. In a later PR, I'd like to move all of these to the builder pattern.